File: workspace-servidor/flotix2021/src/main/java/com/flotix/dto/CaducidadDTO.java
package com.flotix.dto;

/**
 * Objeto que almacena la informacion de las caducidades para su devolucion
 * 
 * @author Flor
 *
 */
public class CaducidadDTO {

	private String id;
	private String idVehiculo;
	private VehiculoDTO vehiculo;
	private String ultimaITV;
	private String proximaITV;
	private String vencimientoVehiculo;
	private boolean baja;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getIdVehiculo() {
		return idVehiculo;
	}

	public void setIdVehiculo(String idVehiculo) {
		this.idVehiculo = idVehiculo;
	}

	public VehiculoDTO getVehiculo() {
		return vehiculo;
	}

	public void setVehiculo(VehiculoDTO vehiculo) {
		this.vehiculo = vehiculo;
	}

	public String getUltimaITV() {
		return ultimaITV;
	}

	public void setUltimaITV(String ultimaITV) {
		this.ultimaITV = ultimaITV;
	}

	public String getProximaITV() {
		return proximaITV;
	}

	public void setProximaITV(String proximaITV) {
		this.proximaITV = proximaITV;
	}

	public String getVencimientoVehiculo() {
		return vencimientoVehiculo;
	}

	public void setVencimientoVehiculo(String vencimientoVehiculo) {
		this.vencimientoVehiculo = vencimientoVehiculo;
	}

	public boolean isBaja() {
		return baja;
	}

	public void setBaja(boolean baja) {
		this.baja = baja;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelDTO/CaducidadDTO.cs

namespace Flotix2021.ModelDTO
{
    class CaducidadDTO
    {
		private string _id;

		public string id
		{
			get { return _id; }
			set { _id = value; }
		}

		private string _idVehiculo;

		public string idVehiculo
		{
			get { return _idVehiculo; }
			set { _idVehiculo = value; }
		}

		private VehiculoDTO _vehiculo;

		public VehiculoDTO vehiculo
		{
			get { return _vehiculo; }
			set { _vehiculo = value; }
		}

		private string _ultimaITV;

		public string ultimaITV
		{
			get { return _ultimaITV; }
			set { _ultimaITV = value; }
		}

		private string _proximaITV;

		public string proximaITV
		{
			get { return _proximaITV; }
			set { _proximaITV = value; }
		}

		private string _venciminetoVehiculo;

		public string venciminetoVehiculo
		{
			get { return _venciminetoVehiculo; }
			set { _venciminetoVehiculo = value; }
		}

		private bool _baja;

		public bool baja
		{
			get { return _baja; }
			set { _baja = value; }
		}
	}
}
